checkST2110.js

```diff
@@ -325,20 +403,69 @@ const test_20_71_1 = (sdp, params) => {
   let streams = sdp.split(/[\r\n]m=/);
   let errors = [];
   if (params.videoOnly) {
-    for ( let s = 1 ; s < streams.length ; s++ ) {
+    for (let s = 1; s < streams.length; s++) {
       if (!streams[s].startsWith('video')) {
         errors.push(new Error(`Stream ${s}: Media type is not 'video' and video only files are in test, as per SMPTE 2110-20 Section 7.1.`));
       }
     }
+  } else {
+    console.log("Test Skipped: Test ST 2110-20 Section 7.1 Test 1 - Use --videoOnly to check for all video.");
   }
+
+  if (params.videoOnly && params.verbose && errors.length == 0)
+    console.log("Test Passed: Test ST 2110-20 Section 7.1 Test 1 - All streams are video");
+
   return errors;
 };
 
+// Test ST 2110-10 Section 6,2 Test 1 - For all video streams, check video params
```

Comment:
Why only video? And `6,2` -> `6.2`.

Reply:
Made the change for 6,2 -> 6.2.  This test is in the original sdpoker. Not sure why there's a test to verify a SDP has video only but only change I made was to instrument it for verbose output.